examples/README.md

```diff
@@ -0,0 +1,99 @@
+
+1. `npm examples:install <version_number>` - will install the dependencies of all the examples matching the `<version_number>`, utilizing the `examples`' internal workspace logic.
+2. `npm run examples:build <version_number>` - will build each code example in the `/examples/<version_number>` directory then copy each example's production output to the `/examples/tmp/<version_number>`. The path to the code example in the `/examples/tmp` follows the [Folder structure](#folder-structure) convention.
+3. `npm run examples:start` - it will start the `http-server` right in the `/examples/tmp` on PORT `8080`. So the URL to the specific code example would be `http://localhost:8080/8.1.0/docs/angular/custom-context-menu/`. The URL follows the same convention as mentioned in the [Live on production](#live-on-production) section.
+
+#### How to run the arbitrary code example
+
+You may want to launch an arbitrary code example instead of a whole folder with all code examples for a specific version. To do that you will need to copy the code example production output to the `/examples/tmp` directory. An example:
+
+To launch only one code example which path is `/examples/8.1.0/docs/angular/custom-context-menu` you must copy the `dist` folder from this example and paste it to the `/examples/tmp/<your_temporary_folder_name>`. After done this, you can run `npm run examples:start` and see your code example under the URL `http://localhost:8080/<your_temporary_folder_name>/`
+
+Also, each example has `npm run start` command which will start a development server. After this command is called visit http://localhost:8080.
+
+### Testing
+
+Each code example has its own test specs, which lives in the `spec` directory.
+
+> Templates have one **failing test**, which has been added to force developers to write passing tests for the next code examples.
+
+#### How to tun tests for arbitrary code example
+
+Each code example has its own test suites which you can run in two ways:
+
+1. Using development server:
+ a. Run the `npm run start` command.
+ b. After the app is launched on a server, run the `npm run jasmine` command which will launch tests specified to this code example.
+2. Running server on the production output folder:
+ a. Run the` npm run test` command which will first build the project then start the `http-server` on the production output folder then launch test specs and finally close the server.
+
+#### How to run tests for the versioned code examples locally
+
+1. Code examples **must** exist in the `examples/<version_number>` directory.
+2. `npm run examples:test <version_number>`. This command will run `npm run test` for each code example (to know what the `npm run test` command does check out the section above).
```

Comment:
This is time consuming. Each test is opening new http-server and closing afterwards? If we have built all examples and copied to `/dist` we can run the server once and then run jasmine for different URLs. Should be much faster